serveradmin/urls.py

```diff
@@ -24,16 +25,17 @@
     url(r'^admin/', admin.site.urls),
 ]
 
-for app in settings.INSTALLED_APPS:
-    module_spec = find_spec(app + '.urls')
+for app in apps.get_app_configs():
+    name = app.name
```

Comment:
This should definitely already work and is the pendant to the previous implementation.
I wonder if we can also make use of the module property of the AppConfigs: https://docs.djangoproject.com/en/1.10/ref/applications/#django.apps.AppConfig.module
Just a mentioning here. Maybe it would be even more reliable in the future.

Reply:
Do you have anything specific in mind ? 

We try to load the urls module which is part of the app module. This might or might not exist - both is valid. I don't see how the app module could help in here or how we can access the urls module from the app module in a way better than this.